src/main/drivers/dshot_bitbang_decode.c

```diff
@@ -100,9 +110,10 @@ static uint32_t decode_bb_value(uint32_t value, uint16_t buffer[], uint32_t coun
     return value;
 }
 
-
+#if defined(STM32F4) || defined(AT32F4)
```

Comment:
This needs a define in target.h instead.

Reply:
Fixed